daemon/clients/base.py

```diff
@@ -34,7 +35,9 @@ def __init__(
         timeout: int = None,
     ):
         self._logger = logger
-        self.timeout = timeout
+        self.timeout = aiohttp.ClientTimeout(
+            timeout if isinstance(timeout, Number) else 10 * 60
```

Comment:
```suggestion
            timeout or 10 * 60
```

Reply:
We need to confirm that it is a number, else default to 10secs

Reply:
but it can be None or a Number as per API, if the called do not respect that is not its fault right?

Reply:
If we allow any value it will throw an error, it's better to avoid that.
```python
async with aiohttp.request(
        method='GET',
        url='https://gorest.co.in/public/v1/users/123/posts',
        timeout=aiohttp.ClientTimeout('abc')
    ) as r:
        print(await r.json())

# TypeError: '>' not supported between instances of 'str' and 'int'
```

Reply:
ok I accept, but I think that if typehimt does not allow str u should assume it does not come, otherwise it is untractable